src/shared/common/components/workstack.jsx

```diff
@@ -39,6 +39,8 @@ const ColumnRenderer = {
 
 const dataAdapters = {
     localDate: (value) => {
+        if (!value || value == '')
+            return '';
         var date = new Date(value);
```

Comment:
looks fine but you could also account for invalid date strings with something like 
```suggestion
        var date = new Date(value);
        if (isNaN(date)) ....
```

Reply:
That seems to have worked. Who knew!?!